client/manifest.json

```diff
@@ -1,8 +1,18 @@
 {
-  "name": "catalog",
-  "short_name": "catalog",
-  "start_url": "/",
-  "background_color": "#6dc38f",
-  "display": "standalone",
-  "theme_color": "#6dc38f"
+    "name": "Web Components",
+    "icons": [
+        {
+            "src": "\/assets\/android-chrome-192x192.png",
```

Comment:
Only used by android chrome I guess. Default names the tool spit out. Not sure anyone else uses this manifest